backend/src/main/java/heartbeat/controller/pipeline/dto/response/PipelineTransformer.java

```diff
@@ -2,18 +2,23 @@
 
 import heartbeat.client.dto.pipeline.buildkite.BuildKitePipelineDTO;
 import heartbeat.client.dto.pipeline.buildkite.StepsDTO;
+import heartbeat.util.GithubUtil;
 
 import java.util.stream.Collectors;
 
 public interface PipelineTransformer {
 
 	static Pipeline fromBuildKitePipelineDto(BuildKitePipelineDTO dto, String orgId, String orgName) {
+		String repositoryFullName = dto.getRepository();
+		String repoName = GithubUtil.getGithubUrlFullName(repositoryFullName);
+
 		return Pipeline.builder()
 			.orgId(orgId)
 			.orgName(orgName)
 			.id(dto.getSlug())
 			.name(dto.getName())
-			.repository(dto.getRepository())
+			.repository(repositoryFullName)
+			.repoName(repoName)
```

Comment:
so confused field `repository` and `repoName`

Reply:
It's not same, and repository is ssh or http url, but repo name is repo name. For example, repository is `git@github.com:au-heartbeat/Heartbeat.git` but repoName is `au-heartbeat/Heartbeat`